tests/test_api_v1.py

```diff
@@ -0,0 +1,19 @@
+# SPDX-License-Identifier: GPL-3.0+
+
+import pytest
+
+from purview.api.v1 import app
+
+
+def pytest_namespace():
```

Comment:
Take a look at WaiverDB for how they handle the pytest fixture:
https://pagure.io/waiverdb/blob/master/f/tests/conftest.py